holoviews/element/raster.py

```diff
@@ -644,6 +421,8 @@ class RGB(Image):
         If an alpha channel is supplied, the defined alpha_dimension
         is automatically appended to this list.""")
 
+    _vdim_reductions = {1: Image}
```

Comment:
There already is an issue about it, so I'll add a comment there, discussing the current state.